help/de/Help_ch_Account-Actions.xml

```diff
@@ -924,16 +1063,17 @@
           persönlichen crontab hinzufügen:
         </para>
 
-        <para><code>0 18 * * 5 /path/to/gnucash --add-price-quotes /path/to/gnucash-filename &gt; /dev/null
-          2&gt;&amp;1</code>
+        <para><code language="cron">0 18 * * 5 /path/to/gnucash --add-price-quotes /path/to/gnucash-filename &gt;
+          /dev/null 2&gt;&amp;1</code>
         </para>
 
         <para>Für &linux;, das dbus verwendet und normalerweise vom Desktop-Manager gestartet wird um mit dconf
           oder gconf zu kommunizieren, lautet der Befehl:
         </para>
 
-        <para><code>0 18 * * 5 env `dbus-launch` sh -c 'trap "kill $DBUS_SESSION_BUS_PID" EXIT; /path/to/gnucash
-          --add-price-quotes /path/to/gnucash-filename &gt; /dev/null 2&gt;&amp;1'</code>
+        <para><code language="cron">0 18 * * 5 env `dbus-launch` sh -c 'trap "kill $DBUS_SESSION_BUS_PID" EXIT;
+          /path/to/gnucash --add-price-quotes /path/to/gnucash-filename &gt; /dev/null
+          2&gt;&amp;1'</code>
         </para>
 
         <para>Denken Sie daran, dass die <quote>Preise</quote> von Investmentfonds in Wirklichkeit der
```

Comment:
Lt. glossary/de.po wollen wir künftig besser zwischen angebotenen `Kurs`en (quote) und realisierten `Preis`en (price) unterscheiden. In der deutschen Übersetztung vom Programm habe ich das irgendwann gemacht. Der Backport in die Sources ist aber noch sehr rudimentär.
In diesem Abschnitt geht es um Kurse –auch wenn im Englischen noch price steht.

Reply:
ok.